styles/index.css

```diff
@@ -0,0 +1,156 @@
+* {
+  box-sizing: border-box;
+  padding: 0 0;
+  color: #fff;
+}
```

Comment:
- [ ] Great job resetting some of the default CSS styles! 🚀 However, please consider also resetting the margin, as of right now, your project has a white margin surrounding the entire content:

![image](https://user-images.githubusercontent.com/77069011/235170565-c7ec4ed5-e575-4273-a04e-3c0aec5009e1.png)

Please consider fixing this! 🤓